apps/site/src/views/Resources/interfaces/interfaces.ts

```diff
@@ -0,0 +1,24 @@
+export interface tag {
+	link: string;
+	src: string;
+	alt: string;
+	className?: string;
+}
+
+export interface Backend_Group_Props {
+	card: string | undefined;
+	title: string | undefined;
+	description: string | undefined;
+	tagList: tag[] | undefined;
```

Comment:
Suggestion: using Hungarian notation here seems unproductive: consider `tags`, or why not just `links`?